app/src/components/misc/PersonaSurvey/types.ts

```diff
@@ -16,35 +16,19 @@ export interface Option {
 }
 
 export interface PageConfig {
-  pageId: number;
+  page: number;
+  pageId: SurveyPage;
   title: string;
   subTitle: string;
-  /**
-   * If skip true then don't show the question in survey
-   */
-  skip?: boolean;
+  skip?: boolean; // If skip true then don't show the page in survey
```

Comment:
nit: Add comment as a doc comment, helps in documentation

https://github.com/requestly/requestly/assets/61556757/8c357c19-9b48-43a1-a6af-86cc8204f60b